src/Console/LintCommand.php

```diff
@@ -17,7 +17,7 @@ public function configure()
     {
         $this
             ->setName('lint')
-            ->addArgument('path', null, 'The path to scan for twig files.', '.')
+            ->addArgument('paths', InputArgument::IS_ARRAY | InputArgument::OPTIONAL, 'The path to scan for twig files.', ['.'])
```

Comment:
Hi, this is a breaking change fonctionnality. You must allow single path and folder without array. I think you can add tests two and update the readme.

Thank you for your contribution

Reply:
Hi,

My change does not break existing functionality, you can still call twigcs with a single path argument.

If wanted I can add a unit-test.

Reply:
Ok, I just verified, it's ok for me ! Nice job !